edgepi-dac.html

```diff
@@ -134,7 +134,7 @@ <h3>Properties</h3>
     <dt>Method</dt>
 		<dd>Select method to either read or write.</dd>
     <dt> Voltage to write<span class="property-type">number</span></dt>
-		<dd>When on write configuration: the amount of voltage between 0 and 10 to write to a selected channel.</dd>
+		<dd>The voltage to write to a selected channel, bound between 0 V and 10 V</dd>
```

Comment:
instead of `Voltage to write` use `Output Voltage` 
and the second line: 
```
The output voltage for the selected channel. Limit 0-5 or 0-10 if gain is enabled.
```

Reply:
this is still not fixed!